core/diag.c

```diff
@@ -583,8 +586,12 @@ void print_rad_types() {
         rad_type_counts[NU_ELECTRON]);
     fprintf(stdout, " ANTI                     %.2f %%\n",
         rad_type_counts[ANTINU_ELECTRON]);
-    fprintf(stdout, " HEAVY                    %.2f %%\n",
+    fprintf(stdout, " X                        %.2f %%\n",
         rad_type_counts[NU_HEAVY]);
+#if RAD_NUM_TYPES > 3
```

Comment:
In decs.h, the condition is `#if RAD_NUM_TYPES >= 4`. Do you want to keep that for consistency here?

Reply:
good idea